near/src/config.rs

```diff
@@ -405,7 +405,7 @@ impl NearConfig {
                 sync_height_threshold: 1,
                 header_sync_initial_timeout: Duration::from_secs(10),
                 header_sync_progress_timeout: Duration::from_secs(2),
-                header_sync_stall_ban_timeout: Duration::from_secs(40),
+                header_sync_stall_ban_timeout: Duration::from_secs(120),
```

Comment:
This means a peer can potentially grieve you for more than two minutes (notice that for the ban to trigger it usually requires entering the loop three times). Not sure whether that is acceptable.

Reply:
Any period of time can grieve by sending rarely information. We can push this into config and allow people to configure this based on their settings.